deep-learning/src/test/scala/com/microsoft/azure/synapse/ml/onnx/ONNXModelSuite.scala

```diff
@@ -396,4 +396,43 @@ class ONNXModelSuite extends TestBase
       .map(wrappedArr => wrappedArr.head.head)
       .toArray
   }
+
+  private lazy val testTextModel: ONNXModel = {
+    spark
+    val model = downloadModel("TfIdfVectorizer.onnx", baseUrl)
+    new ONNXModel()
+      .setModelLocation(model.getPath)
+      .setFeedDict(Map("text" -> "features"))
+      .setFetchDict(Map("encoded" -> "result"))
+  }
+
+  private lazy val testDfText: DataFrame = {
+    Seq(
+      Tuple1(Array("A", "B", "C")),
+      Tuple1(Array("A", "B", "B", "C", "A"))
+    ).toDF("features")
+  }
+
+  test("ONNX model can accept variable size input if batch size is set to 1"){
+    val Array(row1, row2) = testTextModel
+      .setMiniBatchSize(1) // If the array size in each row can vary, must set batch size to 1.
```

Comment:
nit: just a friendly reminder to add this suggestion to the appropriate error message so people know that it is supported if theres no auto-batching :)

Reply:
done.